source/jormungandr/jormungandr/scenarios/helper_classes/helper_utils.py

```diff
@@ -327,7 +290,38 @@ def _build_fallback(
             if not _is_crowfly_needed(
                 pt_obj.uri, fallback_durations, accessibles_by_crowfly.crowfly, fallback_dp
             ):
-                _update_fallback_sections(pt_journey, fallback_dp, fallback_period_extremity, mode)
+                _update_fallback_sections(pt_journey, fallback_dp, fallback_period_extremity)
+
+                # update distances and durations by mode if it's a proper computed streetnetwork fallback
+                if fallback_dp and fallback_dp.journeys:
+                    all_modes = ['walking', 'bike', 'car', 'ridesharing']
```

Comment:
My point was to have a single place where we define all the different street-network's modes once for all. As of now, there is no single source of truth if you want to get those modes. 

I'm not asking to do the change now (it's a much bigger change), but I'm stressing out the point.